plenary/plenary-12.yaml

```diff
@@ -0,0 +1,267 @@
+metadata:
+  title: Resolutions taken at the 12th meeting of ISO/TC 154
+  date: 
+    start: 1996-10-21
+    end: 1996-10-22
+  source: Secretariat of ISO/TC 154
+resolutions:
+  - dates:
+      - 1996-10-21
+      - 1996-10-22
+    identifier: 113
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: resolves
+        date_effective: 
+          start: 1996-10-21
+          end: 1996-10-22
+        message: ISO/TC 154 resolves, at the request of ISO/IEC JTC1 SC 31, to set up a liaison with ISO/IEC JTC1/SC 31
+
+
+  - dates:
+      - 1996-10-21
+      - 1996-10-22
+    identifier: 114
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: regards
+        date_effective: 
+          start: 1996-10-21
+          end: 1996-10-22
+        message: ISO/TC 154 regards the harmonisation and registration of data as one of its major roles and recognises that the BSR is a key tool to achieve that objective. Further, in accordance with the ISO BSR/CC resolution 96/8 stating that the BSR Rules and Guidelines and Content shall become ISO standards with its own registration authority, ISO/TC 154 would like to be given the responsability for processing the BSR Project during the project phase and will require the setting up of a subcommittee and establishing a maintenance agency for ongoing BSR operation and maintenance once the project phase has been completed
+      - type: requests
+        date_effective: 
+          start: 1996-10-21
+          end: 1996-10-22
+        message: ISO/TC 154 also request its members to participate actively in the BSR work
```

Comment:
In this resolution, I used the word `regards` as a type action. Is it correct?

Reply:
This one is a bit tricky, I think this works better:

```yaml
    considerations:
      - type: considering
        date_effective: 2019-10-17
        message: ISO/TC 154 regards the harmonisation and registration of data as one of its major roles and recognises that the BSR is a key tool to achieve that objective.

      - type: according
        date_effective: 2019-10-17
        message: Further, in accordance with the ISO BSR/CC resolution 96/8 stating that the BSR Rules and Guidelines and Content shall become ISO standards with its own registration authority

    actions:
      - type: requests
        date_effective:
          start: 1996-10-21
          end: 1996-10-22
        message: ISO/TC 154 would like to be given the responsibility for processing the BSR Project during the project phase and will require the setting up of a subcommittee and establishing a maintenance agency for ongoing BSR operation and maintenance once the project phase has been completed

      - type: requests
        date_effective:
          start: 1996-10-21
          end: 1996-10-22
        message: ISO/TC 154 also request its members to participate actively in the BSR work
```